Configuration.props

```diff
@@ -174,6 +174,10 @@
     <RemapAssemblyRefTool>$(ManagedRuntime) $(ManagedRuntimeArgs) &quot;$(RemapAssemblyRefToolExecutable)&quot;</RemapAssemblyRefTool>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <UpdateXlfOnBuild Condition="'$(RunningOnCI)' != 'true'">true</UpdateXlfOnBuild>
```

Comment:
What is this property used by?  I see no other references to `UpdateXlfOnBuild` within this PR.

Reply:
Ah yes, whoops. I meant to surface all the notes from the new commit message. For this question, step 3 in particular is the key info.

The [dotnet/xliff-tasks][0] tasks and targets find `.resx` files in a
project, generate or update corresponding `.xlf` [XLIFF][1] files as
needed, and then build satellite assemblies from the `.xlf` files.

Using xliff-tasks involves a few steps:

 1. Add the dotnet-eng Azure DevOps NuGet package feed or the
    dotnet-core MyGet NuGet package feed to `NuGet.config`.  This commit
    uses dotnet-eng because that appears to be the more up-to-date
    recommendation, based on the [dotnet/arcade README][2].

 2. Add the XliffTasks NuGet package to the project to localize.

 3. Set `$(UpdateXlfOnBuild)` to `true` for the project when building
    locally, but not when building on CI.  This follows the strategy
    suggested in the [xliff-tasks README][3]:

        Many teams using `XliffTasks` default `UpdateXlfOnBuild` to true
        for local developer builds, but leave it off for CI builds. This
        way the .xlf files are automatically updated as the developer
        works, and the CI build will fail if the developer forgets to
        include the changes to the .xlf files as part of their PR. This
        way the .xlf files are always in sync with the source files, and
        can be handed off to a localization team at any time.

    This commit sets the property globally via `Configuration.props` to
    help make it easily visible to contributors.

 4. Build the project to generate the initial set of `.xlf` files.

 5. Add the generated `.xlf` files to source control.

 6. Update the targets for the installer packages so that they include
    the satellite assemblies generated by XliffTasks.

In the future, if `Xamarin.Android.Build.Tasks.csproj` is converted to a
short-form project, it would probably be best to add a dependency on
dotnet/arcade, set `$(UsingToolXliff)` to `true`, and then undo steps
1-3 since dotnet/arcade takes care of those steps automatically.

I locally verified that the satellite assemblies were included in the
expected locations in both the Windows and macOS installers with this
commit in place.

I also locally verified that MSBuild used the text from
`Resources.fr.xlf` for the XA4214 warning if I modified
`GenerateJavaStubs.Run()` to set both
`Thread.CurrentThread.CurrentCulture()` and
`Thread.CurrentThread.CurrentUICulture()` to
`new CultureInfo ("fr-FR")`.

[0]: https://github.com/dotnet/xliff-tasks
[1]: http://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html
[2]: https://github.com/dotnet/arcade/blob/2c6db6ee8d8adeb2e8ccc1485e6780635890e419/README.md#getting-started
[3]: https://github.com/dotnet/xliff-tasks/blob/91369f6f0585fa092eddfa6c169b7f68174eb763/README.md#updating-xlf-files